src/containerapp/azext_containerapp/custom.py

```diff
@@ -3345,6 +3345,18 @@ def list_replicas(cmd, resource_group_name, name, revision=None):
                                             revision_name=revision)
 
 
+def count_replicas(cmd, resource_group_name, name, revision=None):
+    app = ContainerAppClient.show(cmd, resource_group_name, name)
```

Comment:
Please use `handle_raw_exception(e)` to handle exceptions.

Reply:
Done.